recipes/remi-php73.rb

```diff
@@ -18,7 +18,7 @@
 
 unless platform?('fedora')
   include_recipe 'yum-epel'
-  include_recipe 'yum-remi-chef::remi-safe'
+  include_recipe 'yum-remi-chef::remi'
```

Comment:
Can you please provide some context on why this was changed? I'm concerned this might cause issues to some folks.

Reply:
I will change it back, i actually thought this was a typo on my part oops